src/Umbraco.Web.UI.Client/src/less/application/grid.less

```diff
@@ -217,19 +223,17 @@ body {
     padding-left:20px
 }
 
-
-
 @media (max-width: 767px) {
 
     // make leftcolumn smaller on tablets
     #leftcolumn {
-      width: 60px;
+        width: 61px;
     }
     #applications ul.sections {
-        width: 60px;
+        width: 61px;
     }
     ul.sections.sections-tray {
-        width: 60px;
+        width: 61px;
```

Comment:
Ahh, I think that was a mistake. It should only ve the margin-left that get 1px added because the ul.sections has 1px right border.